wolox-books/src/app/screens/login/login.component.ts

```diff
@@ -30,6 +30,8 @@ export class LoginComponent implements OnInit {
   }
 
   loginPost() {
-    this.loginService.loginUser(this.loginForm.value).subscribe(responsePost => console.log('Succesfull', responsePost) );
+    this.loginService.loginUser(this.loginForm.value).subscribe(responsePost => { console.log('Succesfull', responsePost)
+    localStorage.setItem("token", JSON.stringify(responsePost))});
```

Comment:
Use single quote marks for strings in typescript, by the way, I think you can remove the console.log at this point